qt/scientific_interfaces/Inelastic/QENSFitting/ParameterEstimation.cpp

```diff
@@ -7,6 +7,7 @@
 #include "ParameterEstimation.h"
 #include "MantidAPI/IFunction.h"
 
+#include <cmath>
 #include <math.h>
```

Comment:
```suggestion
#include <cmath>
```
Should we just have either `cmath` or `math.h`, and not both?